openpype/hosts/flame/plugins/create/create_shot_clip.py

```diff
@@ -23,10 +23,10 @@ def process(self):
                 # nested dictionary (only one level allowed
                 # for sections and dict)
                 for _k, _v in v["value"].items():
-                    if presets.get(_k):
+                    if presets.get(_k, None) is not None:
                         gui_inputs[k][
                             "value"][_k]["value"] = presets[_k]
-            if presets.get(k):
+            if presets.get(_k, None) is not None:
```

Comment:
Shouldn't this be this?
```suggestion
            if presets.get(k, None) is not None:
```

Reply:
Very good catch. Thanks ;)